### ISSUE TITLE
Disk parameters are in the wrong order

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#214]
Currently this is the way we describe the disk

disk.mdot(msol/yr)                            1e-7
Disk.illumination.treatment(0=no.rerad,1=high.albedo,2=thermalized.rerad,3=analytic)                    3
Disk.temperature.profile(0=standard;1=readin)                    0
disk.radmax(cm)                             2.4e+10

These are clearly in the wrong order, since disk.mdot is not needed if we read the temperature profile in

The order should probably be
Disk.temperature.profile(0=standard;1=readin)                    0
disk.mdot(msol/yr)                            1e-7
disk.radmax(cm)                             2.4e+10
Disk.illumination.treatment(0=no.rerad,1=high.albedo,2=thermalized.rerad,3=analytic)                    3

Note also we should decide whether to use upper or lower case to begin 
and that the word analytic is not very descriptive.  This option refers to a case where the disk  T is increased due to illumination from the star

**Comment by kslong** [comment]
More generally, the current set of disk parameters are entered in two different places in the input file.  Why aren't they in a single place?

**Comment by kslong** [comment]
Disk.illum no longer exists.  It has been replaced by Surface.illum and refers to anything including a star.

Today, I tried to address this problem.  I pulled all of the parameters that are needed to define the disk into get_disk_params, so that information about the disk is not longer requested from within python.c and get_disk_params.  I changed some of the names of the disk params so they are more object oriented (so @smangham should be pleased).  I updated synonyms so that old parameter files should work.  I also split get_disk_params out into a separate file setup_disk.c so that it will be easier to locate where these are read in in the future.

This by no means closes all of the issues associated with IO redesign #222 but hopefully it helps.
